### DISCUSSION TITLE
zsh: command not found: markdown-json

### DISCUSSION THREAD

**Discussion by diego-de-sousa** [klaytonfaria/markdown-json#29] [Q&A]
I'm probably missing some sleep or something but I don't know why is not working after `npm install markdown-json`
@klaytonfaria 
What am I doing wrong?

Thanks

**Comment by klaytonfaria** [comment]
Hey @cozarkd ! 
First of all, really sorry for the delay. 🙏🏾 I'm trying to be back more frequently here!
About the question... Did you try to install globally (with the `-g` argument)?

```bash
npm install -g markdown-json
```
